site/docs/master/contributions/minio.md

```diff
@@ -229,6 +229,20 @@ You must also get the Minio URL, which you can then specify as the value of the
 
 1.  Edit your `BackupStorageLocation` YAML, adding `publicUrl: <URL_FROM_PREVIOUS_STEP>` as a field under `spec.config`. You must include the `http://` or `https://` prefix.
 
+## Accessing logs with an HTTPS endpoint
+
+If you're using Minio with HTTPS, you may see unintelligible text in the output of `velero describe`, or `velero logs` commands.
+
+In order to fix this, you can add a public, HTTP URL to the `BackupStorageLocation`, or a [self-signed certificate][32].
```

Comment:
If you're using TLS with minio, is there actually an `http`-only endpoint? My impression was that it was one or the other, not both.

My understanding of this issue was that either (a) the user didn't provide a `spec.publicUrl` that was routable from the end-user's machine; or (b) that they were using a self-signed cert for minio which wasn't previously supported by velero.

My point is, from my understanding, "adding an HTTP URL to the spec.publicUrl" isn't actually a valid option if the user is using TLS. Am I missing something?

Reply:
I just asked for clarification from the user. It's possible that they reconfigured minio to just disable TLS.

I re-read the minio docs, and I think I was misunderstanding. Your understanding sounds correct - minio can't provide both HTTPS and HTTP endpoints at the same time.

My interpretation of the issue was that it was scenario (b), since data was retrieved, it was just unreadable nonsense.

So pending any update from the reporter, I think I should reword the v1.3.2 version to advise users to upgrade to v1.4.0 for (b).